### ISSUE TITLE
No way to distinguish between "Heat/Cool" and "Heat" for HVAC

### ISSUE THREAD

**Issue by mgartin** [nielsfaber/scheduler-component#255]
First, thank you for the great integration!

### Checklist
- [x] I checked for similar existing issues (both open and closed) before posting.
- [x] I am running the latest version.

### Expected behavior
I have a Mitsubishi heatpump that is registered via the [MELCloud](https://www.home-assistant.io/integrations/melcloud) integration. In the Scheduler card it would be good if I could distinguish between "Heat" and "Heat/Cool".

### Actual behavior

Both Heat and Heat/Cool modes are shown as "Heat".

![Screenshot_20230225_185918](https://user-images.githubusercontent.com/25937034/221372851-63a4fb07-5e06-4196-aa1a-62ea09578013.png)

### Steps to Reproduce
Addd a schedule, notice that "Heat" is repeated twice. One of them sets the action "Heat/Cool". The order of the two seems to depend on the language used in the app.

**Comment by mgartin** [comment]
I'd like to add that the MELCloud integration itself does distinguish between Heat and Heat/Cool:

![Screenshot_20230225_191757](https://user-images.githubusercontent.com/25937034/221373249-71af1a22-c9f6-4e63-bc84-54ef6b2c2f42.png)

**Comment by nielsfaber** [comment]
I agree it's not very intuitive this way, they should have different name.

Could you share some context?
What is the difference between operation in mode Heat/Cool and mode Heat with setpoint=22?
Would your device start cooling in mode Heat/Cool f the room becomes 23?
Is this desirable behavior or not useful?

Could you also share the properties of this entity under Developer Tools -> States?
